docs/index.rst

```diff
@@ -1,6 +1,59 @@
-Simple Tabs
+Sphinx Tabs
+***********
+
+Create tabbed content in `Sphinx documentation <http://www.sphinx-doc.org>`_ when building HTML.
+
+Installation
+============
+
+```bash
+pip install sphinx-tabs
+```
+
+To enable the extension in Sphinx, add the following to your conf.py:
+
+```python
+extensions = ['sphinx_tabs.tabs']
+```
+
+If needed, there is a configuration option to allow additional builders to be considered compatible. For example, to add the `linkcheck` builder, add the following to your conf.py:
+
+```python
+sphinx_tabs_valid_builders = ['linkcheck']
+```
+
```

Comment:
you are trying to use markdown in an RST file here 😉 

should be e.g.

```
.. code-block:: bash

   pip install sphinx-tabs

```

Reply:
Ah, I didn't notice, as it renders partially correct on the actual site.
![image](https://user-images.githubusercontent.com/10674555/105637698-b43edf00-5e3c-11eb-81e5-70fd559ea4b9.png)

Reply:
yeh I know, I just notice the extra `` ` `` and though something was a miss

Reply:
🤦🏻‍♂️ I copied that straight from the README without thinking...